Cmdline/Main.cs

```diff
@@ -240,6 +240,11 @@ private static int RunAction(Options cmdline, CommonOptions options, string[] ar
                 case "remove":
                     return (new Remove(user)).RunCommand(manager.CurrentInstance, cmdline.options);
 
+                case "replace":
+                    user.RaiseMessage("Replace command detected in Main");
```

Comment:
Could this `RaiseMessage` be a `log.debug` instead? The other commands don't print analogous messages.

Reply:
Oh, yes, that was supposed to be a `log.debug`.